Godeps/Godeps.json

```diff
@@ -3929,11 +3929,6 @@
 			"Rev": "225eecce40065518f2eabc3b52020365db012384"
 		},
 		{
-			"ImportPath": "k8s.io/kubernetes/cmd/libs/go2idl/args",
```

Comment:
Hmmm... apparently it doesn't look if `Godeps.json` is modified, I wonder if it should, I guess so.